src/code42cli/cmds/search/extraction.py

```diff
@@ -61,12 +63,11 @@ def handle_response(response):
                 events = _get_alert_details(sdk, events)
             except Exception as ex:
                 handlers.handle_error(ex)
-        handlers.TOTAL_EVENTS += len(events)
-        event = None
-        for event in events:
-            output_logger.info(event)
-        if event:
-            last_event_timestamp = extractor._get_timestamp_from_item(event)
+        total_events = len(events)
+        handlers.TOTAL_EVENTS += total_events
+        handlers.EVENTS = events
```

Comment:
This will probably change given the other discussions about separating the handlers, but right now this is eating all but the last page of events, since you're overwriting `handlers.EVENTS` instead of appending them to a list.

Reply:
Changed.